app/models/chargeback_rate_detail.rb

```diff
@@ -36,9 +67,12 @@ def find_rate(value)
 
   def cost(value)
     return 0.0 unless self.enabled?
-    value = 1 if group == 'fixed'
+
+    value = 1.0 if fixed?
+
     (fixed_rate, variable_rate) = find_rate(value)
-    hourly(fixed_rate) + hourly(variable_rate) * value
+
+    hourly(fixed_rate) +  hourly(variable_rate) * value
```

Comment:
@lpichler I found a bug in this calculation when I was trying to verify the value for "Storage allocated Cost". It was showing 0 when it should be $29,760.00 => (1.0 fixed cost * 40gb * 744 hours).

It's incorrect because there should be parens around  `hourly(fixed_rate) +  hourly(variable_rate)`. I made this change locally and got the correct cost:
```{diff}
diff --git a/app/models/chargeback_rate_detail.rb b/app/models/chargeback_rate_detail.rb
index 651249a..f9ac46e 100644
--- a/app/models/chargeback_rate_detail.rb
+++ b/app/models/chargeback_rate_detail.rb
@@ -72,7 +72,7 @@ class ChargebackRateDetail < ApplicationRecord
 
     (fixed_rate, variable_rate) = find_rate(value)
 
-    hourly(fixed_rate) +  hourly(variable_rate) * value
+    (hourly(fixed_rate) + hourly(variable_rate)) * value
   end
 
   def hourly(rate)
```
It looks like this was introduced with the tiered rates feature.

Here are before and after shot of the report so that you have a frame of reference -
Before
![old](https://cloud.githubusercontent.com/assets/76593/19399959/0010f97c-9222-11e6-9433-0b8e2c1704ff.png)

After
<img width="159" alt="new" src="https://cloud.githubusercontent.com/assets/76593/19399961/05238ec0-9222-11e6-8be4-404577dc2056.png">